tools/respecDocWriter.js

```diff
@@ -60,7 +60,12 @@ async function fetchAndWrite(
   src,
   out,
   whenToHalt,
-  { timeout = 300000, disableSandbox = false, debug = false } = {}
+  {
+    timeout = 300000,
+    disableSandbox = false,
+    debug = false,
+    useLocal = false,
```

Comment:
Should this be a string/boolean instead of just boolean? Having a string could allow using a custom URL (say, from unpkg) instead of using local one.

Reply:
I don't have a strong opinion. Whatever you think is best... or leave it for now, and add the custom URL when we need it.